pkg/fixtures/purger_s3.go

```diff
@@ -0,0 +1,45 @@
+package fixtures
```

Comment:
this file should now be called purger_blob instead, shouldn't it?

Reply:
Agreed and Applied